server/routes/llm_config.py

```
@@ -18,9 +18,58 @@
+    """Chat-picker filter: provider mode + prefixes that can actually run."""
+    try:
+        mode = os.getenv("LLM_PROVIDER_MODE") or "direct"
+        return jsonify(
+            {
+                "success": True,
+                "mode": mode,
+                "prefixes": picker_prefixes(mode, os.environ),
+            }
+        ), 200
+    except Exception as e:
+        logger.error(f"Error getting picker config: {e}", exc_info=True)
```

Ruff (0.16.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)